rero_ils/modules/ill_requests/cli.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2019 RERO
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Click command-line interface for ill_request record management."""
+
+from __future__ import absolute_import, print_function
+
+import json
+import random
+
+import click
+from flask.cli import with_appcontext
+
+from rero_ils.modules.ill_requests.api import ILLRequest
+from rero_ils.modules.items.cli import get_locations
+from rero_ils.modules.locations.api import Location
+from rero_ils.modules.patrons.api import Patron
+from rero_ils.modules.utils import get_ref_for_pid
+
+
+@click.command('create_ill_requests')
+@click.option('-f', '--requests_file', 'input_file', help='Request input file.')
+@with_appcontext
+def create_ill_requests(input_file):
+    """Create ILL request for each organisation."""
+    location_pids = get_locations()
+    patron_pids = list(Patron.get_all_pids())
```

Comment:
I think this  way you will allow ill_requests from another ogranisations. I do not know if it is the case in this US or not @iGormilhit ?
If not, I you think you need to filter patrons by organsation as well.

Reply:
@BadrAly @zannkukai If a patron from org A needs a document from another org, he or she needs to create an ILL request as if it were a non RERO ILS network, at least for now. So I think Aly is right.